packages/@webex/plugin-meetings/test/integration/spec/space-meeting.js

```diff
@@ -34,7 +34,7 @@ skipInNode(describe)('plugin-meetings', () => {
   describe('space-meeting', () => {
     let space = null;
 
-    before(() =>
+    before(async () =>
```

Comment:
same as above - why is this needed, how does it fix anything?

Reply:
Removed from next, cherry-picked here.

Reply:
Yes @arun3528 . I raised a PR in next first then cherry-picked that here, there the build was green and locally also everything passed.. 
PR - https://github.com/webex/webex-js-sdk/pull/3501